### PR TITLE
Revise tutorial “Performing a Rolling Update”

### PR DESCRIPTION
**Pull request by Biswajit-Mohapatra2**
fixed:  #39493

Adding some changes that mentioned in the #39493 issue.

Changes at:    https://kubernetes.io/docs/tutorials/kubernetes-basics/update/update-intro/

### CODE REVIEW

**Comment by Biswajit-Mohapatra2** [comment]
@divya-mohan0209 : can you please review it, i made changes according to this #39493 issue.

**Comment by sftim** [comment]
/retitle Revise tutorial “Performing a Rolling Update”

File: content/en/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -29,8 +29,7 @@ <h3>Updating an application</h3>
...
 
-            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable during the update and the maximum number of new Pods that can be created, is one. Both options can be configured to either numbers or percentages (of Pods).
-            In Kubernetes, updates are versioned and any Deployment update can be reverted to a previous (stable) version.</p>
+            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable and can be created during the update, is one. Both options can be configured to either a fixed quantity or percentile of Pods. In Kubernetes, updates are versioned and any Deployment update can be reverted to a previous (stable) version.</p>
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/update/update-intro.html:32]: This change loses something. How about:
```suggestion
            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable during the update is one, and the number of temporary extra Pods that can be created during the update is also one. Both options can be configured to either a fixed quantity or percentile of Pods. In Kubernetes, updates are versioned and any Deployment update can be reverted to a previous (stable) version.</p>
```

**Review by sftim — COMMENTED** [review]
A reluctant LGTM: this PR makes the page better and a bit worse, in different parts of the same (source) line.

/lgtm

**Comment by Biswajit-Mohapatra2** [comment]
hi @sftim, can you please again approve this!

**Comment by sftim** [comment]
@Biswajit-Mohapatra2 , could you rebase against _main_? See https://lwkd.info/2023/20230606#developer-news

This PR isn't passing tests as it stands.

**Comment by Ritikaa96** [comment]
Hi, @Biswajit-Mohapatra2 Please rebase against the main as stated above.

**Comment by a-mccarthy** [comment]
@Biswajit-Mohapatra2 hello, a gentle ping to check if you have time to rebase against the `main` branch so we can continue with this PR. Thanks!

**Comment by sftim** [comment]
No progress for some time

/close

@Biswajit-Mohapatra2 you are welcome to make a new PR with the equivalent changes in it.

**Pull request closed without merging**